docs/content/en/tasks/scale/_index.md

```diff
@@ -188,7 +188,49 @@ The StatefulSet controller manages the deletion of Cassandra pods. It deletes on
 This means for example that `my-k8ssandra-dc1-default-sts-3` will be deleted before `my-k8ssandra-dc1-default-sts-2`.
 {{% /alert %}}
 
+## Bootstrap and decommission order
+
+A K8ssandra cluster will always bootstrap new nodes and decommission existing nodes in a strictly
+deterministic order that strives to achieve balanced racks.
+
+When scaling up, the operator will add new nodes to the racks with the fewest nodes. When scaling
+down, the operator will remove nodes from the racks with the most nodes. If two racks have the same
+number of nodes, the operator will start with the rack that comes first in the datacenter
+definition.
+
+Let's take a look at an example. Suppose we have a 3-node datacenter with the following racks:
+
+* `rack1` with 1 node
+* `rack2` with 1 node
+* `rack3` with 1 node
+
+Now let's scale up to, say, 8 nodes. The operator will bootstrap 5 new nodes, exactly in the
```

Comment:
Neither the v1 nor the v2 docs mention anything about the cleanup operation that is run. We should add some info about that.

Reply:
There is a mention already:

```
After the new nodes are up and running, `nodetool cleanup` should run on all of the nodes except the
new ones to remove keys and data that no longer belong to those nodes. There is no need to do this
manually. The cass-operator deployment, which again is installed with K8ssandra, automatically runs
`nodetool cleanup` for you.
```